notificationservice/notification.go

```diff
@@ -0,0 +1,423 @@
+package notificationservice
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/Appliscale/perun/context"
+	"github.com/Appliscale/perun/mysession"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/olekukonko/tablewriter"
+	"os/user"
+	"strings"
+	"time"
+)
+
+type Connection struct {
+	context *context.Context
+	session *session.Session
+
+	SqsClient          *sqs.SQS
+	sqsQueueOutput     *sqs.CreateQueueOutput
+	sqsQueueAttributes *sqs.GetQueueAttributesOutput
+
+	snsClient *sns.SNS
+	TopicArn  *string
+}
+
+var sinkName = "Perun-sink-"
+
+// Configure AWS Resources needed for pregress monitoring
+func ConfigureRemoteSink(context *context.Context) (err error) {
+	conn := initRemoteConnection(context)
+	snsTopicExists, sqsQueueExists, err := conn.verifyRemoteSinkConfigured()
+
+	if snsTopicExists && sqsQueueExists {
+		context.Logger.Info("Remote sink has already been configured")
+		return
+	} else {
+		shouldSNSTopicBeRemoved := snsTopicExists && !sqsQueueExists
+		err = conn.deleteRemainingSinkResources(shouldSNSTopicBeRemoved, false)
+		if err != nil {
+			println("error deleting up remote sink", err.Error())
```

Comment:
Let's use logger for errors everywhere in that review.

Reply:
I believe our logger does not have a warning type